### PR TITLE
PIM-4719: Asset storage

### PR DESCRIPTION
**Pull request by solivier**

### CODE REVIEW

File: cookbook/product_asset/sftp.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Connect to an sftp server
+-------------------------
+
+Lets assume you already configured an sftp server. You will have to add a new adapter in the oneup_flysystem configuration file, with the sftp credentials.
+Once you created your new adapter you just have to set your new adapter in the filesystems category.
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/bundles/oneup_flysystem.yml
```
Comment (jjanvier) [inline cookbook/product_asset/sftp.rst]: app/config.yml

File: cookbook/product_asset/sftp.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Connect to an Amazon AwsS3v3 server
+-----------------------------------
+
+In order to use the AwsS3v3 adapter, you first need to create a client object defined as a service.
+This version requires you to use the "v4" of the signature.
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/bundles/flysystem_clients.yml
```
Comment (jjanvier) [inline cookbook/product_asset/sftp.rst]: app/config/config.yml

File: cookbook/product_asset/sftp.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                        host: ftp.domain.com
+                        port: ~
+                        username: ~
+                        password: ~
+                        root: ~
+                        timeout: ~
+                        privateKey: ~
+                        permPrivate: ~
+                        permPublic: ~
+    filesystems:
```
Comment (jjanvier) [inline cookbook/product_asset/sftp.rst]: do we need to copy this key?

that sucks, we have to find a way not to

File: cookbook/product_asset/sftp.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+    oneup_flysystem:
+        adapters:
+            ...
+            acme.flysystem_adapter:
+                awss3v3:
+                    client: acme.s3_client
+                    bucket: ~
+                    prefix: ~
+    filesystems:
```
Comment (jjanvier) [inline cookbook/product_asset/sftp.rst]: it not good

the goal is to change ONLY the adapter
you don't have to touch the filesystems part as an integrator

File: cookbook/product_asset/sftp.rst (hunk 3 above)
Comment (solivier) [inline cookbook/product_asset/sftp.rst]: no we don't I can remove this it was only to show what were the available keys

**Pull request closed without merging**